Spurious Safari bot failures due to missing safaridriver

<https://github.com/web-platform-tests/wpt/pull/27247> is failing its checks on Safari instances, due to the following error:

<https://dev.azure.com/web-platform-tests/wpt/_build/results?buildId=59780&view=logs&j=bf8f5026-3d1f-5955-26be-17f9e1ef0bb3&t=f2f2a672-3250-5564-df22-32ed9c3bfd05>
```
2021-01-19T22:56:23.4740330Z ##[section]Starting: Run tests
2021-01-19T22:56:23.4750860Z ==============================================================================
2021-01-19T22:56:23.4751540Z Task         : Command line
2021-01-19T22:56:23.4752120Z Description  : Run a command line script using Bash on Linux and macOS and cmd.exe on Windows
2021-01-19T22:56:23.4752740Z Version      : 2.178.0
2021-01-19T22:56:23.4753230Z Author       : Microsoft Corporation
2021-01-19T22:56:23.4753810Z Help         : https://docs.microsoft.com/azure/devops/pipelines/tasks/utility/command-line
2021-01-19T22:56:23.4754480Z ==============================================================================
2021-01-19T22:56:23.6458390Z Generating script.
2021-01-19T22:56:23.6478500Z Script contents:
2021-01-19T22:56:23.6480220Z ./wpt run --yes --no-pause --no-fail-on-unexpected --no-restart-on-unexpected --affected HEAD^1 --log-wptreport /Users/runner/work/1/a/wpt_report.json --log-wptscreenshot /Users/runner/work/1/a/wpt_screenshot.txt --channel preview safari
2021-01-19T22:56:23.6481520Z ========================== Starting Command Output ===========================
2021-01-19T22:56:23.6519340Z [command]/bin/bash --noprofile --norc /Users/runner/work/_temp/d9984b87-27ee-4cba-9c60-6ca35800c9a1.sh
2021-01-19T22:56:41.6763730Z Traceback (most recent call last):
2021-01-19T22:56:41.6766090Z   File "./wpt", line 34, in <module>
2021-01-19T22:56:41.6767000Z     wpt.main()
2021-01-19T22:56:41.6767970Z   File "/Users/runner/work/1/s/tools/wpt/wpt.py", line 176, in main
2021-01-19T22:56:41.6768930Z     rv = script(*args, **kwargs)
2021-01-19T22:56:41.6769880Z   File "/Users/runner/work/1/s/tools/wpt/run.py", line 836, in run
2021-01-19T22:56:41.6770880Z     wptrunner_kwargs = setup_wptrunner(venv, **kwargs)
2021-01-19T22:56:41.6771880Z   File "/Users/runner/work/1/s/tools/wpt/run.py", line 804, in setup_wptrunner
2021-01-19T22:56:41.6772790Z     setup_cls.setup(kwargs)
2021-01-19T22:56:41.6773730Z   File "/Users/runner/work/1/s/tools/wpt/run.py", line 177, in setup
2021-01-19T22:56:41.6774820Z     self.setup_kwargs(kwargs)
2021-01-19T22:56:41.6775770Z   File "/Users/runner/work/1/s/tools/wpt/run.py", line 612, in setup_kwargs
2021-01-19T22:56:41.6776830Z     raise WptrunError("Unable to locate safaridriver binary")
2021-01-19T22:56:41.6777830Z tools.wpt.run.WptrunError: Unable to locate safaridriver binary
2021-01-19T22:56:41.9445510Z ##[error]Bash exited with code '1'.
2021-01-19T22:56:41.9461290Z ##[section]Finishing: Run tests
```

--- a/dom/historical.html
+++ b/dom/historical.html
@@ -4,6 +4,8 @@
 <script src="/resources/testharnessreport.js"></script>
 <div id=log></div>
 <script>
+// TOUCH
+
 function isInterfaceRemoved(name) {
   test(function() {
     assert_equals(window[name], undefined)

--- a/tools/ci/azure/install_chrome.yml
+++ b/tools/ci/azure/install_chrome.yml
@@ -2,8 +2,9 @@ steps:
 # The conflicting google-chrome and chromedriver casks are first uninstalled.
 # The raw google-chrome-dev cask URL is used to bypass caching.
 - script: |
-    HOMEBREW_NO_AUTO_UPDATE=1 brew cask uninstall google-chrome || true
-    HOMEBREW_NO_AUTO_UPDATE=1 brew cask uninstall chromedriver || true
-    HOMEBREW_NO_AUTO_UPDATE=1 brew cask install https://raw.githubusercontent.com/Homebrew/homebrew-cask-versions/master/Casks/google-chrome-dev.rb
+    HOMEBREW_NO_AUTO_UPDATE=1 brew uninstall --cask google-chrome || true
+    HOMEBREW_NO_AUTO_UPDATE=1 brew uninstall --cask chromedriver || true
+    curl https://raw.githubusercontent.com/Homebrew/homebrew-cask-versions/master/Casks/google-chrome-dev.rb > google-chrome-dev.rb
+    HOMEBREW_NO_AUTO_UPDATE=1 brew install --cask google-chrome-dev.rb
   displayName: 'Install Chrome Dev'
   condition: and(succeeded(), eq(variables['Agent.OS'], 'Darwin'))

--- a/tools/ci/azure/install_firefox.yml
+++ b/tools/ci/azure/install_firefox.yml
@@ -1,6 +1,8 @@
 steps:
 # This is equivalent to `Homebrew/homebrew-cask-versions/firefox-nightly`,
 # but the raw URL is used to bypass caching.
-- script: HOMEBREW_NO_AUTO_UPDATE=1 brew cask install https://raw.githubusercontent.com/Homebrew/homebrew-cask-versions/master/Casks/firefox-nightly.rb
+- script: |
+    curl https://raw.githubusercontent.com/Homebrew/homebrew-cask-versions/master/Casks/firefox-nightly.rb > firefox-nightly.rb
+    HOMEBREW_NO_AUTO_UPDATE=1 brew install --cask firefox-nightly.rb
   displayName: 'Install Firefox Nightly'
   condition: and(succeeded(), eq(variables['Agent.OS'], 'Darwin'))

--- a/tools/ci/azure/install_safari.yml
+++ b/tools/ci/azure/install_safari.yml
@@ -8,7 +8,7 @@ steps:
   condition: eq(variables['safaridriver_diagnose'], true)
 - ${{ if eq(parameters.channel, 'preview') }}:
   - script: |
-      HOMEBREW_NO_AUTO_UPDATE=1 brew cask install tools/ci/azure/safari-technology-preview.rb
+      HOMEBREW_NO_AUTO_UPDATE=1 brew install --cask tools/ci/azure/safari-technology-preview.rb
       # Workaround for `sudo safardriver --enable` not working on Catalina:
       # https://github.com/web-platform-tests/wpt/issues/21751
       mkdir -p ~/Library/WebDriver/

